src/main/resources/liquibase/crop_changelog/24_3_0.xml

```diff
@@ -0,0 +1,466 @@
+        <comment>Add cvterm Plot Number </comment>
+        <sql dbms="mysql" splitStatements="true">
+			set @class = ( SELECT cvterm_id FROM cvterm WHERE name = "Passport" AND cv_id = 1011);
+			
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1010, 'Plant Number', 'Plant Number', null, 0, 0);
+									
+			set @propertyId = (select cvterm_id from cvterm where name = 'Plant Number' AND cv_id = 1010);
+			
+			INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1225, @propertyId,@class );
+			
+        </sql>
+    </changeSet>				
+    
+  
+	<changeSet id="v24.3.0-17" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(*) FROM cvterm WHERE name = "PLANT_NUMBER_AP_text" and cv_id = 1040 ;
+            </sqlCheck>
+        </preConditions>
+        <comment>Add variable for Packing _Status</comment>
+        <sql dbms="mysql" splitStatements="true">
+            set @methodID = (select cvterm_id from cvterm where name = 'Assigned Passport Descriptor' AND cv_id = 1020);
+            set @scaleId = (select cvterm_id from cvterm where name = 'Text' AND cv_id = 1030);
+			set @propertyId = (select cvterm_id from cvterm where name = 'Plant Number' AND cv_id = 1010);
+			
+            INSERT INTO cvterm (cv_id, name, definition, dbxref_id, is_obsolete, is_relationshiptype)
+            VALUES (1040, 'PLANT_NUMBER_AP_text', 'Plant Number', null, 0, 0);
+
+            set @variableId = (select cvterm_id from cvterm where name = 'PLANT_NUMBER_AP_text' AND cv_id = 1040);
+
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1200, @variableId, @propertyId);
+
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1210, @variableId, @methodId);
+
+            INSERT INTO cvterm_relationship(type_id, subject_id, object_id)
+            VALUES (1220, @variableId, @scaleId);
+
+            INSERT INTO cvtermprop(cvterm_id, type_id, value, rank)
+            VALUES (@variableId, 1800, 'Germplasm Passport', 0);
+
+        </sql>
+    </changeSet>	
+	
+	<changeSet id="v24.3.0-18" author="rowena">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="1">
+				SELECT COUNT(*)>0 FROM
+					(SELECT CASE WHEN @group = listid THEN @seq := @seq + 1 ELSE @seq := 1 END AS zentryid, @group := listid AS proj_id,  listid ,gid,lrecid
+					FROM listdata l ORDER BY listid, entryid
+					) AS z
+				LEFT JOIN (
+					SELECT DISTINCT entryid, listid,gid,lrecid  FROM listdata ORDER BY listid, entryid 
+					) AS zz
+				ON z.listid = zz.listid AND z.zentryid  = zz.entryid ;
+            </sqlCheck>
+        </preConditions>
+        <comment>Update listdata with sequence number</comment>
+        <sql dbms="mysql" splitStatements="true">
+		
+				SET @seq := 0; 
```

Comment:
Are the SQL for updating the entry id of listdata and stock performant? Do they only affect the listdata/stock records that are not sequential?